Prac_06/programming_language.py

```diff
@@ -0,0 +1,18 @@
+class ProgrammingLanguage:
+    def __init__(self, name="", typing="", reflection=True, year=0):
```

Comment:
By default, `reflection=True` should be `reflection=""`. This is because a language isn't automatically reflected. `year=0` should also be `year=""` :)